packages/jest-expo/jest-preset.js

```diff
@@ -27,15 +27,55 @@ jestPreset.transform['\\.[jt]sx?$'] = [
   { caller: { name: 'metro', bundler: 'metro', platform: 'ios' } },
 ];
 
-const defaultAssetNamePattern = '^.+\\.(bmp|gif|jpg|jpeg|mp4|png|psd|svg|webp)$';
-if (!jestPreset.transform[defaultAssetNamePattern]) {
-  console.warn(`Expected react-native/jest-preset to define transform[${defaultAssetNamePattern}]`);
-} else {
-  delete jestPreset.transform[defaultAssetNamePattern];
-}
+/* Update this when metro changes their default extensions */
+const defaultMetroAssetExts = [
```

Comment:
not ideal to in-line this but it's consistent with how we [emulate the behavior of our metro config](https://github.com/expo/expo/blob/04dcaa3cba2b465c5865468b05df410a4c2662bd/packages/jest-expo/jest-preset.js#L10-L15) in this preset